docs/content/comparisons/druid-vs-kudu.md

```diff
@@ -5,15 +5,18 @@ layout: doc_page
 Druid vs Kudu
 =============
 
-Kudu's storage format enables single row updates, whereas updates to existing Druid segments requires recreating the segment, hence,   
-the process for updating old values is slower in Druid. The requirements in Kudu for maintaining extra head space to store 
+Kudu's storage format enables single row updates, whereas updates to existing Druid segments requires recreating the segment, so theoretically    
+the process for updating old values should be slower in Druid. However, the requirements in Kudu for maintaining extra head space to store 
```

Comment:
clarify that "slower" means "higher latency" – "slow" is too vague since it could refer to either latency or throughput